src/ros_tcp_endpoint/default_server_endpoint_venv.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python
+
+import os
+import subprocess
+import sys
+from time import sleep
+
+def is_venv():
+    return (hasattr(sys, 'real_prefix') or
+            (hasattr(sys, 'base_prefix') and sys.base_prefix != sys.prefix))
+
+arg='--venv'
```

Comment:
nit: Could you fix the formatting in this PR? The [pre-commi](https://github.com/Unity-Technologies/ROS-TCP-Endpoint/runs/4217867319?check_suite_focus=true)t github action has a list of formatting suggestions.